pkgs/development/libraries/vtk/default.nix

```diff
@@ -30,7 +39,18 @@ stdenv.mkDerivation rec {
   # At least, we use -fPIC for other packages to be able to use this in shared
   # objects.
   cmakeFlags = [ "-DCMAKE_C_FLAGS=-fPIC" "-DCMAKE_CXX_FLAGS=-fPIC" ]
-    ++ optional (qtLib != null) [ "-DVTK_USE_QT:BOOL=ON" ];
+    ++ optional (qtLib != null) [ "-DVTK_USE_QT:BOOL=ON" ]
+    ++ optional stdenv.isDarwin [ "-DBUILD_TESTING:BOOL=OFF"
+                                  "-DCMAKE_OSX_SYSROOT="
+                                  "-DCMAKE_OSX_DEPLOYMENT_TARGET=" ];
+
+  doCheck = !stdenv.isDarwin;
+
+  patchPhase = stdenv.lib.optionalString stdenv.isDarwin ''
```

Comment:
Please consider using `postPatch`, overriding the standard patch phase breaks e.g., `patches` (not used here, but still).